### ISSUE TITLE
Error when calling district offices which have a dash

### ISSUE THREAD

**Issue by jlev** [OpenSourceActivismTech/call-power#100]
traceback

```
ValueError: too many values to unpack
(5 additional frame(s) were not displayed)
...
  File "call_server/call/views.py", line 573, in make_single
    (current_target, created) = Target.get_or_create(uid, prefix)
  File "call_server/campaign/models.py", line 248, in get_or_create
    data = check_political_data_cache(key, cache)
  File "call_server/political_data/data_cache.py", line 8, in check_political_data_cache
    adapted_key, adapter_suffix = adapter.key(key)

ValueError: too many values to unpack
```

**Comment by jlev** [comment]
Fixed with 51425f8d05617a23f85d7d0d8196afdc1fa875af

**Comment by jlev** [comment]
Sentry issue: [CALLPOWER-CREDO-6J](https://sentry.io/spacedog/callpower-credo/issues/856672286/?referrer=github_plugin)
